src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -1221,8 +1223,10 @@ public IEnumerable<ReplayProgress> Replay(Stream replayStream)
 
                         var cmd = DeserializeCommand(strType, item, context, peepingTomStream);
 
+                        //Todo To remove check after all MergedTransactionCommand driven class include in switch
```

Comment:
`TODO:`

Reply:
And what check are you talking about?

Reply:
Check for null.
Now it should serialize all the commands or throw error